go.mod

```diff
@@ -1,12 +1,26 @@
 module github.com/canonical/jimm-go-sdk/v3
 
-go 1.21
+go 1.22.5
```

Comment:
We need to stick with Go 1.21 for now.
See https://github.com/canonical/jimm-go-sdk/pull/5